build-system/babel-config/replace-plugin.js

```diff
@@ -18,6 +18,7 @@
 const argv = require('minimist')(process.argv.slice(2));
 const experimentsConfig = require('../global-configs/experiments-config.json');
 const experimentsConstantBackup = require('../global-configs/experiments-const.json');
+const {getExperimentConstant} = require('./helpers');
```

Comment:
Thank you for adding `helpers.js` -- I agree with the idea behind it. However, I just realized that `replace-plugin.js` is basically a helper file, so for better maintainability, could you rename `replace-plugin.js` to `helpers.js` and add `getExperimentConstant()` to the same file?